src/components/core/a11y/focus.ts

```diff
@@ -16,8 +19,11 @@ export const IS_FOCUSABLE_QUERY = [
 // Note: the use of this function for more complex scenarios (with many nested elements) may be expensive.
 export function getFocusableElements(
   elements: HTMLElement[],
-  filterFunc?: (el: HTMLElement) => boolean,
-  findFirstFocusable?: boolean,
+  properties?: {
+    filterFunc?: (el: HTMLElement) => boolean;
+    findFirstFocusable?: boolean;
+    includeNotVisibleElements?: boolean;
```

Comment:
Does maybe read better?
```suggestion
    includeInvisibleElements?: boolean;
```